lib/paylocity_web_service/connection.rb

```diff
@@ -95,6 +95,7 @@ def connection
         conn.use Faraday::Request::Authorization, :Bearer, access_token
         conn.request :retry, max: 2
         conn.request :json
+        conn.request :instrumentation
```

Comment:
Hi @mayankdedhia 

The code looks good to me. 

Can we add a more precise name for the events generated by paylocity ruby gem?

for example:

```
conn.request :instrumentation, name: 'request.paylocity'
```

It will much easier to filter out paylocity events when subscribing to events.

Reply:
Added `request.paylocity` as the instrumentation name.